meson.build

```diff
@@ -0,0 +1,24 @@
+project(
+  'h2non-semver-c', 'c',
```

Comment:
Is it needed or convenient to use `h2non-` as package prefix?

Reply:
See my comment below ')